beatport_importer.user.js

```diff
@@ -95,65 +90,66 @@ function retrieveReleaseInfo(release_url, release_data) {
         }
-        tracks.push({
+        mbtracks.push({
             artist_credit: MBImport.makeArtistCredits(artists),
             title: title,
-            duration: track.duration.minutes,
+            duration: track.length,
         });
     });
 
-    let unique_artists = [];
-    $.each(release_artists, function (i, el) {
+    const unique_artists = [];
+    $.each(release_artists, (index, el) => {
         if ($.inArray(el, unique_artists) === -1) {
             unique_artists.push(el);
         }
     });
 
     if (unique_artists.length > 4) {
-        release.artist_credit = [MBImport.specialArtist('various_artists')];
+        mbrelease.artist_credit = [MBImport.specialArtist('various_artists')];
     } else {
-        release.artist_credit = MBImport.makeArtistCredits(unique_artists);
+        mbrelease.artist_credit = MBImport.makeArtistCredits(unique_artists);
     }
-    release.discs.push({
-        tracks: tracks,
-        format: release.format,
+
+    mbrelease.discs.push({
+        tracks: mbtracks,
+        format: mbrelease.format,
     });
 
-    LOGGER.info('Parsed release: ', release);
-    return release;
+    LOGGER.info('Parsed release: ', mbrelease);
+    return mbrelease;
 }
 
 // Insert button into page under label information
-function insertLink(release, release_url, isrcs) {
-    let edit_note = MBImport.makeEditNote(release_url, 'Beatport');
-    let parameters = MBImport.buildFormParameters(release, edit_note);
+function insertLink(mbrelease, release_url, isrcs) {
+    const edit_note = MBImport.makeEditNote(release_url, 'Beatport');
+    const parameters = MBImport.buildFormParameters(mbrelease, edit_note);
 
-    let mbUI = $(
-        `<li class="interior-release-chart-content-item musicbrainz-import">${MBImport.buildFormHTML(
+    const mbUI = $(
+        `<div class="interior-release-chart-content-item musicbrainz-import">${MBImport.buildFormHTML(
             parameters
-        )}${MBImport.buildSearchButton(release)}</li>`
+        )}${MBImport.buildSearchButton(mbrelease)}</div>`
     ).hide();
 
     $(
         '<form class="musicbrainz_import"><button type="submit" title="Submit ISRCs to MusicBrainz with kepstin’s MagicISRC"><span>Submit ISRCs</span></button></form>'
     )
         .on('click', event => {
-            const query = isrcs.map(({ isrc, number }) => (isrc == null ? `isrc${number}=` : `isrc${number}=${isrc}`)).join('&');
+            const query = isrcs.map((isrc, index) => (isrc == null ? `isrc${index + 1}=` : `isrc${index + 1}=${isrc}`)).join('&');
             event.preventDefault();
             window.open(`https://magicisrc.kepstin.ca?${query}`);
         })
         .appendTo(mbUI);
 
-    $('.interior-release-chart-content-list').append(mbUI);
+    $('div[title="Collection controls"]').append(mbUI);
     $('form.musicbrainz_import').css({ display: 'inline-block', 'margin-left': '5px' });
     $('form.musicbrainz_import button').css({ width: '120px' });
```

Comment:
The import button alignment was off in my browser (Brave on Linux). This may have been the case before the redesign but I don't think so:

![image](https://github.com/murdos/musicbrainz-userscripts/assets/59319/60e275a5-081d-4dbf-b335-e48c3173bd19)

In any case the following change fixed it for me:

```suggestion
    $('form.musicbrainz_import button').css({ width: '120px' });
    $('form.musicbrainz_import button img').css({ display: 'inline-block' });
```

Reply:
This issue occurred after the redesign and I was wondering what to do about it. Thank you for your suggestion!